build/npm/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@getgauge/cli",
+  "description": "npm wrapper for installing gauge command line",
+  "scripts": {
+    "install": "node ./src/index.js",
+    "test": "mocha"
+  },
+  "bin": {
+    "gauge": "./bin/gauge"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/getgauge/gauge.git"
+  },
+  "keywords": [
+    "getgauge",
+    "testing",
+    "gauge"
+  ],
+  "author": "getgauge",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/getgauge/gauge/issues"
+  },
+  "homepage": "https://github.com/getgauge/gauge#readme",
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "mocha": "^5.2.0",
+    "sinon": "^6.2.0"
+  },
+  "dependencies": {
+    "request": "^2.88.0",
+    "unzipper": "^0.9.3"
+  }
+}
```

Comment:
Missing `veriosn` field in `package.json` file which is required for installation.

Reply:
see PR [description](https://github.com/getgauge/gauge/pull/1179#issue-214841855).

The `version` is kept out because it needs to be set. This is to ensure that the version is set via the pipeline. Otherwise there needs to be a commit for every deploy.

I am trying to avoid a commit from the pipeline.

Reply:
Got it.  Make sense.